packages/genesys-spark-components/src/components/stable/gux-tooltip/gux-tooltip.tsx

```diff
@@ -124,7 +125,8 @@ export class GuxTooltip {
     }).then(({ x, y, middlewareData, placement }) => {
       Object.assign(this.root.style, {
         left: `${x}px`,
-        top: `${y}px`
+        top: `${y}px`,
+        visibility: middlewareData.hide?.referenceHidden ? 'hidden' : 'visible'
```

Comment:
@gavin-everett-genesys in popup you used 
```
        referenceHidden
          ? this.popupElementContainer.classList.add('gux-sr-only-clip')
          : this.popupElementContainer.classList.remove('gux-sr-only-clip');
```
Do you remember why you picked that over `visibility` (if there was a reason)?

Reply:
Using the `visibility:hidden` will hide the popup visually from the screen reader. So if the popup goes out of view the focus would be lost from the popup. If we use the `gux-sr-only-clip` the popup will also be hidden but be still accessible to screen-reader.

Reply:
I don't think the tooltip in this case is focusable, so would the popup reasoning apply here?

Reply:
No I think this PR is fine as is if that was the reasoning for popup.